src/terra-dev-site/doc/application/reference.1/components/ApplicationNavigation.doc.mdx

```diff
@@ -0,0 +1,46 @@
+import PropsTable from 'terra-application/lib/application-navigation/ApplicationNavigation?dev-site-props-table';
+import ApplicationNavigationExample from 'terra-application/lib/terra-dev-site/doc/application/reference.1/components/example/ApplicationNavigationExample?dev-site-example';
+
+# ApplicationNavigation
+
+The ApplicationNavigation component provides a responsive layout and a variety of controls for application navigation.
+
+## Usage
```

Comment:
we should drop the usage section since we have the code example they can reference

Reply:
The 'Usage' section for the components here really just show how to import the components. I think that can still be helpful face-up for people vs. viewing the example code.